lib/scholar/decomposition/pca.ex

```diff
@@ -471,7 +471,7 @@ defmodule Scholar.Decomposition.PCA do
     end
   end
 
-  defnp flip_svd(u, v) do
+  defn flip_svd(u, v) do
```

Comment:
It is ok to make it public but let's `@doc false` it so it does not appear on docs.

```suggestion
  @doc false
  defn flip_svd(u, v) do
```